### PR TITLE
Appliance: Update Backup/Restore + Upgrading Docs

### PR DESCRIPTION
**Pull request by khorne3**

### CODE REVIEW

File: articles/appliance/admin/updating-the-appliance.md [hunk 1]
```diff
@@ -5,8 +5,58 @@ description: How to update the Appliance
 
 # Updating the Appliance
 
-Auth0 requires regular updates to your Appliance instances to ensure that you always have the latest features and improvements. The update interval is typically once per month.
+To ensure that your Appliance has the latest functionality, security, and bug fixes, Auth0 requires you to perform regular updates. While the Auth0 engineering team releases updates on a monthly basis, you should plan updates on a monthly, bi-monthly, and quarterly basis. You should not exceed **90 days** without updating the Appliance.
 
-During the update, the Appliance instances will need limited connectivity to the Internet to download the application and operating system updates). The update will be [started via the Management Dashboard](/appliance/dashboard/updates) and typically takes between 10 to 30 minutes.
+::: panel Appliance Releases
```
Comment (mpaktiti) [inline articles/appliance/admin/updating-the-appliance.md]: Change the panels to notes, the text is short.

File: articles/appliance/admin/updating-the-appliance.md [hunk 2]
```diff
@@ -5,8 +5,58 @@ description: How to update the Appliance
...
+
+* Take a VM snapshot of each node you're updating.
+* Ensure that you've enabled access to your nodes for the Auth0 Customer Success Engineer who will be assisting you with the update.
+* Complete the pre-check test **one hour** prior the start of the update.
+  * Please be sure that your infrastructure engineer has administrative access to the Dashboard (specifically the Root Tenant Authority).
+  * Ensure that all [Health Checks](/appliance/dashboard/troubleshoot#health-check) are okay.
+
+  ::: panel Monitoring
+  For additional information on gathering testing information, please see [Appliance Monitoring](/appliance/monitoring).
+  :::
```
Comment (mpaktiti) [inline articles/appliance/admin/updating-the-appliance.md]: I think you should remove the indentation, this doesn't looks very nice.

image

File: articles/appliance/cli/backing-up-the-appliance.md [hunk 3]
```diff
@@ -5,46 +5,62 @@ description: How to back up the Appliance using its CLI
...
 ## Prior to Beginning the Backup
 
-Please ensure that your workstation has been [configured to perform operations](/appliance/cli/configure-cli) on your Appliance instances;
+Please ensure that:
+* You have configured the [Command Line Interface](/appliance/cli/configure-cli) on your Appliance instances;
+* The node has disk space equal to or greater than twice the amount of Auth0 data present.
 
-::: note
-  Please perform backups only on Appliance instances running **build 7247** or later. Backups are only allowed on nodes added to the `backup` role. Please see [Adding a Node to the Backup Role](/appliance/cli/adding-node-to-backup-role) for additional information.
+::: panel Backup Role
```
Comment (mpaktiti) [inline articles/appliance/cli/backing-up-the-appliance.md]: Revert back to note? Use always notes or warnings, unless you have a block of text (4+ lines).

File: articles/appliance/cli/backing-up-the-appliance.md [hunk 4]
```diff
@@ -60,24 +76,89 @@ If a backup is available, you will see a message similar to the following:
...
+## Delete a Backup
 
 To delete an existing backup, you will use the "backup-delete" message in your local command-line instance:
 
 `a0cli -t <target node> backup-delete`
 
-## Restoring a Backup
+## Restore a Backup
+
+::: panel-warning Sensitive Configuration Information
```
Comment (mpaktiti) [inline articles/appliance/cli/backing-up-the-appliance.md]: Use `warning`, no panel needed.

File: articles/appliance/cli/backing-up-the-appliance.md [hunk 5]
```diff
@@ -5,46 +5,62 @@ description: How to back up the Appliance using its CLI
 
 # How to Back Up Appliance Instances Using the CLI
```
Comment (mpaktiti) [inline articles/appliance/cli/backing-up-the-appliance.md:7]: Long doc, add toc maybe?

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Some minor changes, mainly related to UI elements.

**Comment by khorne3** [comment]
@mpaktiti Fixed. Thanks!

**Pull request merged**